insights/specs/default.py

```diff
@@ -267,6 +271,7 @@ def httpd_cmd(broker):
 
     @datasource(ipcs_s)
     def semid(broker):
+        """Command - semids"""
```

Comment:
Should this be `Command:` to be consistent with the others?

Reply:
Oh yea, I changed format at one point and missed that one. Thanks Bob, good catch.